concourse/pipelines/windows-image-build.jsonnet

```diff
@@ -298,6 +299,138 @@ local containerimgbuildjob = {
   ],
 };
 
+local windowsinstallmediaimgbuildjob = {
+  local job = self,
+
+  image:: error 'must set image in windowsinstallmediaimgbuildjob',
+  workflow:: error 'must set workflow in windowsinstallmediaimgbuildjob',
+
+  // Start of job.
+  name: 'build-' + job.image,
+  on_success: {
+    task: 'publish-success-metric',
+    config: common.publishresulttask {
+      pipeline: 'windows-image-build',
+      job: job.name,
+      result_state: 'success',
+      start_timestamp: '((.:start-timestamp-ms))',
+    },
+  },
+  on_failure: {
+    task: 'publish-failure-metric',
+    config: common.publishresulttask {
+      pipeline: 'windows-image-build',
+      job: job.name,
+      result_state: 'failure',
+      start_timestamp: '((.:start-timestamp-ms))',
+    },
+  },
+  plan: [
+    { get: 'compute-image-tools' },
+    { get: 'guest-test-infra' },
+    {
+      task: 'generate-timestamp',
+      file: 'guest-test-infra/concourse/tasks/generate-timestamp.yaml',
+    },
+    {
+      load_var: 'start-timestamp-ms',
+      file: 'timestamp/timestamp-ms',
+    },
+    {
+      task: 'generate-build-id',
+      file: 'guest-test-infra/concourse/tasks/generate-build-id.yaml',
+      vars: { prefix: job.image },
+    },
+    {
+      put: '%s-gcs' % job.image,
+      params: { file: 'build-id-dir/%s*' % job.image },
+    },
+    {
+      load_var: 'gcs-url',
+      file: '%s-gcs/url' % job.image,
+    },
+    {
+      task: 'get-secret-iso-path-2022',
+      config: gcp_secret_manager.getsecrettask { secret_name: 'win2022-64' },
+    },
+    {
+      load_var: 'iso_path_2022',
+      file: 'gcp-secret-manager/' + 'win2022-64',
```

Comment:
For these variables, they can be a single string instead of two string that are concatenated.

https://github.com/GoogleCloudPlatform/guest-test-infra/blob/1e1879aa7dbcde153218a1f6f80d0d2c1e3dbcf6/concourse/pipelines/windows-image-build.jsonnet#L97 is an example where the rest of this file uses a single string.

'gcp-secret-manager/' + 'win2022-64' --> 'gcp-secret-manager/win2022-64'

Same for the other secret manager variables when it's a set string.

Reply:
done